addons/sourcemod/scripting/ttt/ttt_taser.sp

```diff
@@ -64,46 +67,50 @@ public Plugin myinfo =
-	
+
 	LoadTranslations("ttt.phrases");
-	
+
 	BuildPath(Path_SM, g_sConfigFile, sizeof(g_sConfigFile), "configs/ttt/config.cfg");
 	Config_Setup("TTT", g_sConfigFile);
 	g_iCreditsTaserHurtTraitor = Config_LoadInt("ttt_hurt_traitor_with_taser", 2000, "The amount of credits an innocent or detective will recieve for discovering a traitor with their zues/taser.");
-	
+	g_bAddSteamIDtoLogs = Config_LoadBool("ttt_steamid_add_to_logs", true, "Should we add steam id to all log actions? Prevent abusing with namefakers");
+	g_iSteamIDLogFormat = Config_LoadInt("ttt_steamid_log_format", 1, "Which steam id format to you prefer? 1 - SteamID2 (STEAM_1:1:40828751), 2 - SteamID3 ([U:1:81657503]) or 3 - SteamID64/CommunityID (76561198041923231)");
+
 	Config_LoadString("ttt_plugin_tag", "{orchid}[{green}T{darkred}T{blue}T{orchid}]{lightgreen} %T", "The prefix used in all plugin messages (DO NOT DELETE '%T')", g_sPluginTag, sizeof(g_sPluginTag));
-	
+
 	Config_Done();
-	
+
 	BuildPath(Path_SM, g_sConfigFile, sizeof(g_sConfigFile), "configs/ttt/taser.cfg");
 	Config_Setup("TTT-Taser", g_sConfigFile);
-	
+
 	g_iDPrice = Config_LoadInt("ta_detective_price", 9000, "The amount of credits a taser costs as detective. 0 to disable.");
 	g_iIPrice = Config_LoadInt("ta_innocent_price", 9000, "The amount of credits a taser costs as innocent. 0 to disable.");
 	g_iTPrice = Config_LoadInt("ta_traitor_price", 0, "The amount of credits a taser costs as traitor. 0 to disable.");
-	
+
 	g_iDPrio = Config_LoadInt("ta_detective_sort_prio", 0, "The sorting priority of the taser (Detective) in the shop menu.");
 	g_iIPrio = Config_LoadInt("ta_innocent_sort_prio", 0, "The sorting priority of the taser (Innocent) in the shop menu.");
 	g_iTPrio = Config_LoadInt("ta_traitor_sort_prio", 0, "The sorting priority of the taser (Traitor) in the shop menu.");
-	
+
 	g_iDCount = Config_LoadInt("ta_detective_count", 1, "The amount of usages for tasers per round as detective. 0 to disable.");
 	g_iICount = Config_LoadInt("ta_innocent_count", 1, "The amount of usages for tasers per round as innocent. 0 to disable.");
 	g_iTCount = Config_LoadInt("ta_traitor_count", 1, "The amount of usages for tasers per round as traitor. 0 to disable.");
-	
+
 	g_iDamage = Config_LoadInt("ta_damage", 0, "The amount of damage a taser deals for detectives and innocents");
 	g_iTDamage = Config_LoadInt("ta_traitor_damage", 0, "The amount of damage a taser deals for traitors");
-	
+
 	g_bOnSpawn = Config_LoadBool("ta_give_taser_spawn", true, "Give the Detective a taser when he spawns?");
 
 	g_bBroadcastTaserResult = Config_LoadBool("ta_broadcast_taser_result", false, "When set to true the results of the taser message will be printed to everyone instead of the client that tased");
-	
+
 	g_bInflictor = Config_LoadBool("ta_barrel_fix", true, "Prevent bug with taser and a explosive barrel");
-	
+
 	Config_Done();
 	
+	g_hOnTased = CreateGlobalForward("TTT_OnTased", ET_Ignore, Param_Cell, Param_Cell);
```

Comment:
You should add AskPluginLoad2 for this (like https://github.com/Bara20/TroubleinTerroristTown/blob/master/addons/sourcemod/scripting/ttt/ttt_knockout.sp#L56).

Reply:
Done, haven't used github this extensively so not sure if I did that right.